tests/testthat/test-pagination.R

```diff
@@ -451,9 +471,20 @@ test_that("rep_cols works as intended for listings and tables", {
   expect_true(grepl(out[51], pattern = "vs")) # vs is repeated
   expect_true(grepl(out[51], pattern = "gear")) # gear is repeated
 
+  ## XXX this doesn't make sense, key cols are always repeated columns
+  ## having a listing with specified key columns and then
+  ## saying the number of rep columns is something else is an anti-pattern
+  ## we should not support. I made it work to not remove a test in my PR, but we should remove this test
```

Comment:
I understand what you mean, but as far as we do not impose it manually (i.e. we add a stop with error or a warning) I would test it, so if we change something we need to go back and change the test. Is not suggested but until we do not block this behavior we might want to keep the test. I heard of people wanting to change the sorting of keycols so this can come useful at a certain point

Reply:
They can change the sorting of key columns by changing the which columns are key columns, or we can allow them to set a sorting order for key columns, but we really don't want to be in the business of having two completely separate partially overlapping concepts of which columns are repeated

Reply:
It makes sense! We can remove the test at a certain point. I would also add some checks to disallow it if we do not want to support it